Sources/UInt128/UInt128.swift

```diff
@@ -591,27 +591,42 @@ extension UInt128 : CustomStringConvertible {
 
     // MARK: Instance Methods
 
-    /// Converts the stored value into a string representation.
-    /// - parameter radix:
-    ///     The radix for the base numbering system you wish to have
-    ///     the type presented in.
-    /// - parameter uppercase:
-    ///     Determines whether letter components of the outputted string will be in
-    ///     uppercase format or not.
-    /// - returns:
-    ///     String representation of the stored UInt128 value.
+    /// Converts the stored value into a string representation based on radix.
+    /// - Parameters:
+    ///   - radix: The radix for the base numbering system to emit. Default is 10.
+    ///   - uppercase: Specify true for uppercase and false for lowercase  for returned String.
```

Comment:
Minor whitespace issue:
```suggestion
    ///   - uppercase: Specify true for uppercase and false for lowercase for returned String.
```